compiler/src/iree/compiler/Codegen/LLVMGPU/ROCDLSelectLoweringStrategy.cpp

```diff
@@ -21,7 +22,11 @@ class ROCDLSelectLoweringStrategyPass
     : public ROCDLSelectLoweringStrategyBase<ROCDLSelectLoweringStrategyPass> {
 public:
   void getDependentDialects(DialectRegistry &registry) const override {
-    registry.insert<IREE::Codegen::IREECodegenDialect>();
+    // clang-format off
```

Comment:
Is this needed actually? Don't think we involve transform dialect stuff in this pull request?

Reply:
I got
> LLVM ERROR: Building op `bufferization.alloc_tensor` but it isn't known in this MLIRContext: the dialect may not be loaded or this operation hasn't been added by the dialect.

without adding it.

Reply:
You need the bufferization dialect. Not the transform dialect.

Reply:
@MaheshRavishankar Thanks!